.golangci.yml

```diff
@@ -72,3 +72,131 @@ linters-settings:
     allow-leading-space: true
     require-explanation: true
     require-specific: false
+  gosimple:
+    # Sxxxx checks in https://staticcheck.io/docs/configuration/options/#checks
+    # Default: ["*"]
+    checks: ["all,-S1034"]
+
+  gocritic:
+    # Which checks should be enabled; can't be combined with 'disabled-checks'.
+    # See https://go-critic.github.io/overview#checks-overview.
+    # To check which checks are enabled run `GL_DEBUG=gocritic golangci-lint run`.
+    # By default, list of stable checks is used.
+    #    enabled-checks:
+    #      - nestingReduce
+    #      - unnamedResult
+    #      - ruleguard
+    #      - truncateCmp
+    # Which checks should be disabled; can't be combined with 'enabled-checks'.
+    # Default: []
+    disabled-checks:
+      - regexpMust
+      - ifElseChain
+      - singleCaseSwitch
+      - appendAssign
+      - typeSwitchVar
+    # Enable multiple checks by tags, run `GL_DEBUG=gocritic golangci-lint run` to see all tags and checks.
+    # See https://github.com/go-critic/go-critic#usage -> section "Tags".
+    # Default: []
+#    enabled-tags:
```

Comment:
Why are we adding so many comments? Can we delete them?

Reply:
sure thing